### PR TITLE
Prepare for 1.3.0

### PR DESCRIPTION
**Pull request by RedMickey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 1.3.0
+* Enhanced creation of debugging configurations. Now debugging configurations are generated interactively. You just need to click on `React Native` button in the IntelliSense prompt in `launch.json` file and then select required parameters in selection panels [#1468](https://github.com/microsoft/vscode-react-native/pull/1468)
+* Added a notification with a link to the CHANGELOG after the extension update [#1476](https://github.com/microsoft/vscode-react-native/pull/1476)
+* Improved async functions control while deleting paths recursively on starting the extension [#1481](https://github.com/microsoft/vscode-react-native/pull/1481)
```
Comment (JiglioNero) [inline CHANGELOG.md:4]:
```suggestion
* Fixed error deleting paths recursively on the extension activation [#1481](https://github.com/microsoft/vscode-react-native/pull/1481)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+* Improved extension security [#1472](https://github.com/microsoft/vscode-react-native/pull/1472)
+* Updated documentation [#1477](https://github.com/microsoft/vscode-react-native/pull/1477), [#1469](https://github.com/microsoft/vscode-react-native/pull/1469)
```
Comment (JiglioNero) [inline CHANGELOG.md:6]:
```suggestion
* Enhance React Native for Windows and macOS documentation [#1477](https://github.com/microsoft/vscode-react-native/pull/1477), [#1469](https://github.com/microsoft/vscode-react-native/pull/1469)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+* Added a notification with a link to the CHANGELOG after the extension update [#1476](https://github.com/microsoft/vscode-react-native/pull/1476)
+* Fixed the bug with incorrect recursive deletion of paths when activating the extension [#1481](https://github.com/microsoft/vscode-react-native/pull/1481)
+* Improved extension security [#1472](https://github.com/microsoft/vscode-react-native/pull/1472)
+* Enhanced contributing guide and documentation about debugging React Native for Windows and macOS applications [#1477](https://github.com/microsoft/vscode-react-native/pull/1477), [#1469](https://github.com/microsoft/vscode-react-native/pull/1469)
+* Internal changes:
+    * Fixed ESLint issues in the code and added Prettier support [#1463](https://github.com/microsoft/vscode-react-native/pull/1463), [#1475](https://github.com/microsoft/vscode-react-native/pull/1475)
```
Comment (etatanova) [inline CHANGELOG.md:8]:
```suggestion
* Fixed ESLint issues in the code and added Prettier support [#1463](https://github.com/microsoft/vscode-react-native/pull/1463), [#1475](https://github.com/microsoft/vscode-react-native/pull/1475)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+
 ## 1.2.0
 * Added `Run React Native LogCat Monitor` and `Stop React Native LogCat Monitor` command palette commands. These commands allow to control LogCat outputs monitoring and create monitors for Android devices on demand. See [more information here](https://github.com/microsoft/vscode-react-native#configure-an-android-logcat-monitor) [#1461](https://github.com/microsoft/vscode-react-native/pull/1461)
```
Comment (etatanova) [inline CHANGELOG.md:11]:
```suggestion
* Added `Run React Native LogCat Monitor` and `Stop React Native LogCat Monitor` palette commands. These commands allow to control LogCat outputs monitoring and create monitors for Android devices on demand. See [more information here](https://github.com/microsoft/vscode-react-native#configure-an-android-logcat-monitor) [#1461](https://github.com/microsoft/vscode-react-native/pull/1461)
```

File: CHANGELOG.md (hunk 3 above)
Comment (RedMickey) [reply CHANGELOG.md:8]: This indent is required for correct display of internal changes. Let's leave it unchanged.

**Review by etatanova — APPROVED** [review]

**Review by JiglioNero — APPROVED** [review]

**Pull request merged**